src/blocks/recommendations/components/parsely-recommendations-inspector-controls.tsx

```diff
@@ -26,30 +27,36 @@ interface ParselyRecommendationsInspectorControlsProps {
 const ParselyRecommendationsInspectorControls = ( {
 	attributes: { boost, imagestyle, limit, openlinksinnewtab, showimages, sort, title },
 	setAttributes,
-} : ParselyRecommendationsInspectorControlsProps ) => (
-	<InspectorControls>
+} : ParselyRecommendationsInspectorControlsProps ) => {
+	function setImageStyle( value: string ): void {
+		setAttributes( {
+			imagestyle: value === 'original' ? 'original' : 'thumbnail',
+		} );
+	}
```

Comment:
As per SonarCloud we should avoid using the functions directly as props because of possible extra renders and either use a function defined in the class or utilise `useCallback` for function.

In rest of the places we used useCallback approach but in imagestyle case we can't use it because we were using it inside conditional so therefore defining the class level function

SonarCloud Rule: https://sonarcloud.io/organizations/parsely/rules?open=typescript%3AS6480&rule_key=typescript%3AS6480